packages/www/src/pages/index.tsx

```diff
@@ -186,9 +183,9 @@ export default function Homepage() {
           <div class="info sticky">
             <h2>🌐 Full i18n support</h2>
             <p>
-              Brisa has a built-in internationalization (i18n) support that
-              allows you to <b>translate</b> your pages and <b>routing</b>,
-              carrying only the translations you consume.
+              Brisa has built-in internationalization support that allows you to{" "}
```

Comment:
```suggestion
              Brisa has built-in internationalization (i18n) support that allows you to{' '}
```

Reply:
Why do you think we need to say the same thing again? Isn't i18n just an abbreviation of internationalization?

Reply:
This mainly for clarity, I understand your point as well, and I agree. Maybe we could do a Reference page or footer and list there all abbreviations?

The point of having it explained somewhere is because some people won't know the naming convention. Because they decide to learn coding with Brisa, because they didn't have to use it before, etc. And we do not have it unified enough, some places we call it i18n and other internationalization. It is for that the reinforcement on this one.

But maybe good idea to go with your suggestion here and write in another place the definition / references.